### PR TITLE
Done Changing the profile icon #1646

### PR DESCRIPTION
**Pull request by Ashmitha2606**
### Pull Request Checklist

- [x] I have added screenshots and videos to show before and after the working of my code.
- [x] I have ensured that the screen size is set to 100% while making the video.
- [x] I have synced the latest fork with my local repository and resolved any conflicts.
- [ ] I have mentioned the issue number which I created before making this PR .(format to mention issue number is : fixes #(issue number) ) 
- [x] I understand that if any the above conditions are not met , my PR will not be MERGED .

### CODE REVIEW

**Comment by ANSHIKA-26** [comment]
its not working , on clicking it nothing happens , kindly fix

File: index.html [hunk 1]
```diff
@@ -386,10 +386,11 @@ <h2>WordWise</h2>
                 <div class="dropdown">
                     <!-- Profile Icon -->
     
-                        <a href="javascript:void(0);" class="profile-link" style="display: flex; align-items: center; text-decoration: none;" onclick="toggleMenu()">
-                            <img src="https://i.postimg.cc/NFFmtc9K/xnz-Mst5-V-400x400.jpg" alt="Profile Icon" style="width: 50px; height: 50px; border-radius: 50%; margin-top: -17px;">
-                        </a>
-                    </li>
+                    <a href="#" class="profile-icon" id="profileDropdown" onclick="toggleDropdown()">
```
Comment (ANSHIKA-26) [inline index.html:389]: keep the anchor tag , just change the style , if you change the class and onClick event ,it will change the js code too

**Review by ANSHIKA-26 — CHANGES_REQUESTED** [review]

**Comment by Ashmitha2606** [comment]
@ANSHIKA-26 i have done the changes please see #1646

**Comment by Ashmitha2606** [comment]
@ANSHIKA-26 hey hi  #1644 i have changed this issue too can you award this one seperatelyy

**Comment by ANSHIKA-26** [comment]
can you please follow PR checklist and open a new PR ,? i need before and after changes pictures , a clear description of what you did and issue numbers , also you need to sync fork , thanks

**Comment by Ashmitha2606** [comment]
Ok sure mill make a new pr and send

**Pull request closed without merging**